### DISCUSSION TITLE
Is one cache-file can be be shared between different smart-dns instances with different interfaces?

### DISCUSSION THREAD

**Discussion by GektorUA** [pymumu/smartdns#1806] [Q&A]
Subj, for example:

instance 1:
...
cache-file /tmp/smartdns.cache
server 8.8.8.8:53 -interface eth2
...

instance 2:
...
cache-file /tmp/smartdns.cache
server 8.8.8.8:53 -interface eth3
...

**Comment by PikuZheng** [comment]
To avoid dns pollution I recommend not to do this.

yes. it works for the same version, configuration group, and domain group.
the cache file is only read once at startup.
